src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -876,6 +876,9 @@ private void AddPatchNode(Patch patch, Vector2 position)
 
         node.Enabled = patchEnableStatusesToBeApplied?[patch] ?? true;
 
+        if (patch.ActiveDisplayVisuals.Contains(WorldEffectVisuals.UnderwaterVentEruption))
```

Comment:
I think the code here should be made extensible to allow a wide variety of events to be added in the future. It's probably okay to not support multiple events at once in a single patch.

Reply:
I can do that but are you sure it's good idea to avoid multiple events?

Reply:
I mean per patch. If there's like 3 events in a vents patch showing all of them in the patch map would make the icon super busy and the player would need to spend quite long looking at the map. So I think it probably makes better gameplay that each patch can have only one event affect it per generation.